src/topology/algebra/uniform_group.lean

```diff
@@ -230,6 +230,54 @@ variables {G}
 ⟨λ h u hu, h _ ⟨u, hu, λ _, id⟩, λ h v ⟨u, hu, hv⟩ x, exists_imp_exists (by exact λ a,
   exists_imp_exists (λ ha hp, mem_of_superset hp (λ i hi a ha, hv (by exact hi a ha)))) ∘ h u hu x⟩
 
+ @[to_additive] lemma topological_group.tendsto_uniformly_on_mul
```

Comment:
Is it true that for a commutative topological group, the multiplication map `G × G → G` is uniformly continuous?

If so, can we factor this result through that?

Reply:
Looks like uniform continuity is true (proof below). However, I'm having trouble seeing how to make use of it.
```
@[to_additive] lemma topological_group.tendsto_uniformly_mul :
  @uniform_continuous₂ G G G
    (topological_group.to_uniform_space G)
    (topological_group.to_uniform_space G)
    (topological_group.to_uniform_space G) (*) :=
begin
  rintros v ⟨u, hu, huv⟩,
  obtain ⟨w, hw, x, hx, h⟩ := mem_nhds_prod_iff.mp (mem_map.mp
    (continuous_mul.tendsto' ((1 : G), (1 : G)) (1 : G) (one_mul (1 : G)) hu)),
  rw uniformity_prod_eq_prod,
  rw map_map,
  refine mem_prod_iff.mpr _,
  refine ⟨_, preimage_mem_comap hw, _, preimage_mem_comap hx, _⟩,
  rw [set.prod_preimage_eq], -- rfl
  intros a ha,
  refine huv _,
  refine (congr_arg (∈ u) (div_mul_comm _ _ _ _)).mp (h ha),
end
```

Reply:
My idea was that you could combine that with the following (which also seems to be missing) to get your result:
```lean
lemma tendsto_uniformly.comp' {α β γ ι : Type*} [uniform_space β] [uniform_space γ]
  {F : ι → α → β} {f : α → β} {p : filter ι} {g : β → γ} (hg : uniform_continuous g)
  (h : tendsto_uniformly F f p) :
  tendsto_uniformly (λ i, g ∘ (F i)) (g ∘ f) p :=
begin
  rw [uniform_continuous, filter.tendsto_def] at hg,
  exact λ u hu, by simpa using h _ (hg u hu),
end
```
Hopefully this works!

Reply:
Quickly looking now you may also need to fill in gaps about uniform convergence of product maps but if this approach works, it's the right proof and we'll benefit from adding all these bits and pieces.